crates/cli/src/bin/wasm-bindgen-test-runner/main.rs

```diff
@@ -32,6 +32,16 @@ enum TestMode {
     Worker { no_modules: bool },
 }
 
+struct TmpDirDeleteGuard {
+    path: PathBuf,
+}
+
+impl Drop for TmpDirDeleteGuard {
+    fn drop(&mut self) {
+        let _ = fs::remove_dir_all(&self.path);
```

Comment:
Can we log the error here instead of discarding it?

Reply:
`eprintln!` is fine?

Reply:
I saw that you are using env_logger, will use log then.